src/style.css

```
@@ -0,0 +1,87 @@
+body {
+  margin: 0;
+}
+
+.conteiner {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  flex-direction: row;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-basis: 300px;
+  flex-grow: 1;
+  height: 300px;
+}
+
+@media (min-width: 900px) {
+  .block--4 {
```

```suggestion
  .block--4 {
```
замість цього можна було б використати :nth-child